zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/metrics/CamundaExporterMetrics.java

```diff
@@ -81,6 +83,11 @@ public void recordProcessInstancesArchived(final int count) {
     processInstancesArchived.increment(count);
   }
 
+  // counts the number of operations in the batch.
```

Comment:
:x: Comments should be for why, not what/how. One exception is if the how is quite complex (for good reasons). But in this case, I don't think this comment is useful. 

:wrench: Since we have both Operations and BatchOperations in Operate, I would propose we rename this method `batchOperationsArchived`.